server/routes.go

```diff
@@ -507,6 +507,8 @@ func ListModelsHandler(c *gin.Context) {
 				Digest:     digest,
 				ModifiedAt: info.ModTime(),
 			})
+		} else {
+			models = []api.ModelResponse{}
```

Comment:
Ideally models is initialized as an empty list. This involves updating line 484 to call `make(...)` instead, e.g.

```go
	models := make([]api.ModelResponse, 0)
```